pkg/tests/ossm/smm_test.go

```diff
@@ -0,0 +1,85 @@
+package ossm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestSMM(t *testing.T) {
+	NewTest(t).Id("T39").Groups(Full).Run(func(t TestHelper) {
+		t.Log("This Suite verifies SMM and SMMR behaviors")
```

Comment:
What exactly does it verify?  

BTW: why isn't this test in a separate PR?

Reply:
because I move both at the same time only for that are in the same PR, but the next time will be separated